jbosstools/org.jboss.tools.central.discovery.earlyaccess/plugin.xml

```diff
@@ -48,6 +48,38 @@ appropriate tests + report issues to plug-in providers as required.</description
          <icon image32="images/angularjs_32.png"></icon>
          <overview url="https://marketplace.eclipse.org/content/angularjs-eclipse"></overview>
       </connectorDescriptor>
+      
+      <!-- Windup -->
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.e.migration"
+            certificationId="org.jboss.tools.discovery.certification.experimental.earlyaccess"
```

Comment:
This should probably be certification.supported.earlyaccess, not experimental... unless you think your stuff is REALLY broken and subject to massive breaking API changes. Given you're on version 3.1 not 0.1, I would expect you're fairly stable and not breaking API willy-nilly. :D